build/IPDK_Container/scripts/os_ver_details.sh

```diff
@@ -0,0 +1,50 @@
+#Copyright (C) 2021 Intel Corporation
+#SPDX-License-Identifier: Apache-2.0
+
+#!/bin/bash
```

Comment:
The [shebang line](https://en.wikipedia.org/wiki/Shebang_(Unix)) should be the first line of the shell script

Reply:
fixed in the scripts I have modified.